concent_api/core/views.py

```diff
@@ -624,9 +728,33 @@ def handle_send_force_subtask_results_response(request, client_message):
     client_public_key = decode_client_public_key(request)
 
     if isinstance(client_message.subtask_results_accepted, message.tasks.SubtaskResultsAccepted):
-        client_message_task_id = client_message.subtask_results_accepted.task_to_compute.compute_task_def['task_id']
+        client_message_task_id    = client_message.subtask_results_accepted.task_to_compute.compute_task_def['task_id']
+        client_message_subtask_id = client_message.subtask_results_accepted.task_to_compute.compute_task_def['subtask_id']
+        task_to_compute           = client_message.subtask_results_accepted.task_to_compute
+        report_computed_task      = None
```

Comment:
This won't work. `update_subtask()` won't set `Subtask.report_computed_task` to `None`. If will just ignore the parameter.

Reply:
Yes, that is exactly what I want to do here. It is just needed to pass it to single function call below.